addon/webextension/background/senderror.js

```diff
@@ -57,7 +57,7 @@ window.errorpopup = (function () {
         message = error.message;
       }
     }
-    chrome.notifications.create(id, {
+    browser.notifications.create(id, {
```

Comment:
As mentioned in the other occurrence of `browser.notifications.create`, repeatedly triggering this function may lead to weird results. This might be the root cause of #2335. I'd say debouncing would be a good idea here.

Also need to wrap this in a `watchPromise`.